test/testwindow.go

```diff
@@ -112,6 +112,10 @@ func (w *testWindow) SetOnClosed(closed func()) {
 	w.onClosed = closed
 }
 
+func (w *testWindow) SetWillClose(_ func()) {
+	// no-op
```

Comment:
This should probably mirror the other windows so comply with the WillClose contract

Reply:
The test windows would be easier to test as well, to define the behaviour in some code...